barretenberg/cpp/src/barretenberg/transcript/origin_tag.hpp

```diff
@@ -14,7 +14,7 @@
 
 #define STANDARD_TESTING_TAGS /*Tags reused in tests*/                                                                 \
     const size_t parent_id = 0;                                                                                        \
-    const auto clear_tag = OriginTag();                                                                                \
+    [[maybe_unused]] const auto clear_tag = OriginTag();                                                               \
```

Comment:
Should the remain `[[maybe_unused]]`?

Reply:
yeah this is totally unrelated but master currently does not build without this